### PR TITLE
enhance meta server; remove unwrap()

### PR DESCRIPTION
**Pull request by bartliu827**
# Which issue does this PR close?

Related #.

# Rationale for this change

# What changes are included in this PR?

# Are there any user-facing changes?

### CODE REVIEW

**Comment by lutengda** [comment]
begin benchmark

**Comment by lutengda** [comment]
begin benchmark

**Comment by roseboy-liu** [comment]
https://cnosdb-admin.teleport.cnosdb.com/benchmark-hits/hits_54c3f29.html
https://cnosdb-admin.teleport.cnosdb.com/benchmark-tsbs/tsbs_54c3f29.html

**Comment by lutengda** [comment]
begin benchmark

**Comment by roseboy-liu** [comment]
https://cnosdb-admin.teleport.cnosdb.com/benchmark-hits/hits_54c3f29.html
https://cnosdb-admin.teleport.cnosdb.com/benchmark-tsbs/tsbs_54c3f29.html

**Comment by lutengda** [comment]
begin benchmark

**Comment by roseboy-liu** [comment]
https://cnosdb-admin.teleport.cnosdb.com/benchmark-hits/hits_54c3f29.html
https://cnosdb-admin.teleport.cnosdb.com/benchmark-tsbs/tsbs_54c3f29.html

**Comment by lutengda** [comment]
begin benchmark

**Comment by roseboy-liu** [comment]
https://cnosdb-admin.teleport.cnosdb.com/benchmark-hits/hits_54c3f29.html
https://cnosdb-admin.teleport.cnosdb.com/benchmark-tsbs/tsbs_54c3f29.html

File: meta/src/client.rs [hunk 1]
```diff
@@ -20,7 +21,7 @@ pub type WriteError =
 pub struct MetaHttpClient {
     inner: Arc<reqwest::Client>,
     addrs: Vec<String>,
-    pub leader: Arc<Mutex<String>>,
```
Comment (roseboy-liu) [inline meta/src/client.rs:23]: use RwLock from `parking_lot` crate

**Review by roseboy-liu — APPROVED** [review]

**Review by roseboy-liu — APPROVED** [review]

**Pull request merged**